docs/docs/start/overview.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: Overview
+sidebar_position: 0
+sidebar_custom_props:
+  icon: TbBolt
+---
+import ThemedImage from '@theme/ThemedImage';
+
+# Twenty Documentation
+
+Twenty is a modern CRM offering the flexibility of open source, advanced features, and a sleek design that puts teams in full control. 
+
+Twenty's codebase is adaptable, fostering a collaborative environment where every user can contribute. This extensible and modular design ensures our product evolves with you, cultivating mutually beneficial growth that promotes progress and innovation.
```

Comment:
I'm sorry I know we wrote all of this text in README (probably I committed it), but I really don't like it anymore haha. 
I'll send you things we can take inspiration from in Discord